yew/src/virtual_dom/vnode.rs

```diff
@@ -31,6 +32,35 @@ pub enum VNode {
     VRef(Node),
 }
 
+impl From<&VNode> for Node {
```

Comment:
@totorigolo this is a slight modification of your `reference` method. I'm not sure I love the ergonomics of using the `From` trait so I'm open to changing. I needed this because we don't have `next_sibling` when mounting a component. We need to get it from the ancestor.